phases/bitrise_yml.go

```diff
@@ -217,7 +274,7 @@ func getBitriseYML(searchDir string, inputReader io.Reader) (models.BitriseDataM
 			return models.BitriseDataModel{}, "", fmt.Errorf("failed to get current branch, error: %s", err)
 		}
 
-		branchName, err := askBranch(branch.tracking, inputReader)
+		branchName, err := askBranch(branch.tracking)
```

Comment:
could you please pass me the relevant log?

Reply:
There is a different issue than in the original comment:
- Selecting No for 'Run the scanner on the current branch' is not working. checkBranch contains bufio.NewReader instead of promptui, did not check if it is different in logging style.
```
CHOOSE ACCOUNT
Selected account: laszlo.pusok

SET PRIVACY OF THE PROJECT
Selected privacy: Private

SCANNING GIT REPOSITORY
Remote URL: git@github.com:bitrise-io/bitrise-add-new-project.git
Selected repository: https://github.com/bitrise-io/bitrise-add-new-project.git

SETUP BITRISE.YML
Found bitrise.yml in current directory.
Run the scanner to generate a new bitrise.yml
The current branch is: unify-logging (tracking: origin unify-logging).
Run the scanner on the current branch: No

[15:45:34] Running scanners:
```

- In case of using existing bitrise.yml a different function (askBranch) is called with slightly different behaviour from (checkBranch), This solves the issue of not referring to the scanner, but duplicates code:
```
CHOOSE ACCOUNT
Selected account: laszlo.pusok

SET PRIVACY OF THE PROJECT
Selected privacy: Private

SCANNING GIT REPOSITORY
Remote URL: git@github.com:bitrise-io/bitrise-add-new-project.git
Selected repository: https://github.com/bitrise-io/bitrise-add-new-project.git

SETUP BITRISE.YML
Found bitrise.yml in current directory.
Use the bitrise.yml found in the current directory or specify manually
✔ Enter the path of your bitrise.yml file (you can also drag & drop the file here): /Users/lpusok/Develop/go/src/github.com/biEnter the path of your bitrise.yml file (you can also drag & drop the file here)/Users/lpusok/Develop/go/src/github.com/bitrise-io/bitrise-add-new-project/bitrise.yml
Which branch would you like to be the default?unify-logging
selected workflow: test
```

Reply:
fixed wrong no selection